Find-and-Replace/Replace-text-within-bookmark-content/.NET-Standard/Replace-text-within-bookmark-content/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.Drawing;
+using System.IO;
+
+namespace Replace_text_within_bookmark_content
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Open the file as a stream.
+            using (FileStream inputStream = new FileStream(Path.GetFullPath(@"../../../Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Load the file stream into a Word document.
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
+                {
+                    //Replace a text within the bookmark.
+                    ReplaceBookmarkText(document, "Adventure");
+                    ReplaceBookmarkText(document, "Test");
+                    //Create a file stream.
+                    using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../Sample.docx"), FileMode.Create, FileAccess.ReadWrite))
+                    {
+                        //Save the Word document to the file stream.
+                        document.Save(outputFileStream, FormatType.Docx);
+                    }
+                }
+            }
+        }
+        public static void ReplaceBookmarkText(WordDocument document, string bookmarkName)
```

Comment:
Please modify the method to use in generic manner.

ReplaceBookmarkText(WordDocument, bookmarkName, textToFind, textToReplace)